index.html

```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
 </div>
   </section>
 
-  <div class="section-cards">
+  <div>
+    <div class="project section-cards">
+      
+    </div>
+  </div>
+  <!-- <section class="popup1">
+    <div class="head-img">
+      <h3>Multi Post Stories</h3>
+    <img src="./images/Close.png" alt="exy">
+    </div>
+    <ul>
+      <li>html</li>
+      <li>Bootstrap</li>
+      <li>Ruby on Rails</li>
+    </ul>
+    <div class="img-head">
+      <img src="./images/popup.png" alt="brunet">
+      <p>
+        Lorem Ipsum is simply dummy text of 
+        the printing and typesetting industry. 
+        Lorem Ipsum has been the industry's standard
+        dummy text ever since the 1500s, when an unknown
+        printer took a galley of type and scrambled it to
+        make a type specimen book. It has survived not only
+        five centuries, but also the leap into electronic 
+        typesetting, remaining essent
+      </p>
+    </div>
+    <div class="button-pop">
+      <button><a href="#"></a> See Live<span><img src="./images/popup1.svg" alt="popupbutton"></span></button>
+      <button><a href="#"></a>See source<span><img src="./images/Github1.png" alt="github"></span></button>
+    </div>
+  </section> -->
+  
+  <!-- <div class="section-cards">
```

Comment:
- [ ] This file contains a huge piece of commented code.  Kindly remove it so the repo becomes cleaner 💪